### PR TITLE
greedy_1124

### PR DESCRIPTION
**Pull request by Lisa-Tera**
0~10까지 완료했습니다

### CODE REVIEW

File: greedy_1124/09_20300.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+'''
+import sys
+n = int(input())
+loss = list(map(int, input().split()))
+loss.sort()
+
+if n % 2 == 0:#짝수
+    m = 0
+    for i in range(n//2):
+        m = max(t[i]+t[n-i-1], m)
```
Comment (Gaeun-Kwon) [inline greedy_1124/09_20300.py:31]: 변수 t가 loss 인가요? 코드 정리가 안 된 것 같아요

저는 홀수인 경우 제일 큰 값을 빼고, 대칭되는 값끼리 더한 것 중 max를 구했는데
은서님은 홀짝을 나누어서 홀수인 경우 끝 값을 index 설정으로 제외하셨다는 게 다른 점 같습니다

File: greedy_1124/0_14916.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
+#https://www.acmicpc.net/problem/14916
+#거스름돈 2x+5y =input의 최소 개수 if 불가 : -1 출력
+#입력 14 | 5원 2개  2원 2개 | 출력: 4
+#입력 13 | 5원 1개  2원 3개 | 출력: 5
```
Comment (Gaeun-Kwon) [inline greedy_1124/0_14916.py:4]: 저랑 흐름은 비슷하신데
저는 우선 5로 최대한 나눈 후, 나머지가 홀수면 다시 돌아가는 방식으로 했는데
은서님은 케이스를 나누어 진행하신 것 같습니다

File: greedy_1124/10_20365.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+RB = str(input())
+R_count = RB.count('R')
+B_count = RB.count('B')
+
+#BRRBRBRBRRBB
+colors = { 'B' : 0, 'R' : 0 } 
+colors[RB[0]] +=1 #모두 B로 색칠 
+for i in range(1, n): 
+    if RB[i] != RB[i-1]:#색 다르면 해당 색의 칠하는 횟수 +1
+        colors[RB[i]]+=1
```
Comment (Gaeun-Kwon) [inline greedy_1124/10_20365.py]: 만약 색은 달라졌는데, 그게 B라면(처음에 칠해둔 색과 같다면) 한번 더 색칠하게 되는 건가요?

File: greedy_1124/1_1343.py [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+입력 : 첫째 줄에 크기가 최대 50인 보드판이 주어진다.
+출력 : 첫째 줄에 사전순으로 가장 앞서는 답을 출력한다. 만약 덮을 수 없으면 -1을 출력한다.
+
+예제 입력           예제 출력
+XXXXXX              AAAABB
+XX.XX               BB.BB
+XXXX....XXX.....XX  -1
+X                   -1
+'''
```
Comment (Gaeun-Kwon) [inline greedy_1124/1_1343.py:15]: 저는 경우의 수를 다 나눠서 복잡하게 구현했는데, 수진님과 은서님처럼 replace로 하면 코드도 짧고 그리디 알고리즘이 되겠네요!!
배워갑니당..

File: greedy_1124/4_11399.py [hunk 5]
```diff
@@ -0,0 +1,23 @@
+#https://www.acmicpc.net/problem/11399
+#ATM앞에 N명의 사람들이 돈을 인출하는데 걸리는 시간의 최솟값 구하기
+#if , P1 = 3, P2 = 1, P3 = 4, P4 = 3, P5 = 2 
+# 1 2 3 4 5로 정렬
+
```
Comment (Gaeun-Kwon) [inline greedy_1124/4_11399.py:5]: 저는 mun_list를 reverse sort 해서 해당 인덱스 값을 곱해주는 방식으로 구현했는데, 이렇게 하면 time_list를 따로 선언하지 않아도 돼서 메모리 사용을 줄일 수 있을 것 같아요!

File: greedy_1124/5_11508.py [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+
+#입력
+n = int(input())
+
+li = [int(input()) for _ in range(n)]
+li.sort(reverse=True)#가격 낮춰야하므로 높은 금액부터 
+tot =0
+for i in range(n//3):#2+1이므로 입력된 수 /3 만큼만 돌리기
+    tot += li[i*3] +li[i*3+1]#가격 높은 2개 
+if n % 3 >0:#만약 정수값이 아니고
```
Comment (Gaeun-Kwon) [inline greedy_1124/5_11508.py:17]: 만약 가격 높은 2개를 tot에 더하는 게 아닌, 세번째에 나오는 가격 낮은 1개를 tot에서 빼는 식으로 구현한다면 예외처리 해줄 필요가 없어서 편할 것 같습니다!

File: greedy_1124/7_13305.py [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+
+n = int(input())
+li = list(map(int, input().split()))
+cost = list(map(int, input().split()))
+
+
+tot =0
+
+for i in range(n-1):# 5 2 4
+    if i ==0:  c=cost[i]
```
Comment (Gaeun-Kwon) [inline greedy_1124/7_13305.py:14]: 매번 i==0인지 체크하는 것 보다, for문 들어가기 전에 c=cost[0]으로 초기화하는 게 더 좋을 것 같습니다!
전체적인 흐름은 제 코드랑 같으시네요(저는 다른 분들 풀이를 참고했습니다 ㅠㅠ ㅎㅎ)

File: greedy_1124/09_20300.py (hunk 1 above)
Comment (Lisa-Tera) [reply greedy_1124/09_20300.py:31]: 제가 코드를 계속 바꾸다가 정리가 제대로 되지 않은 코드를 올렸나보내요! 수정하겠습니다!

File: greedy_1124/7_13305.py (hunk 7 above)
Comment (Lisa-Tera) [reply greedy_1124/7_13305.py:14]: 네! 참고하겠습니다!!
